### ISSUE TITLE
(Feature Request) Import / Export

### ISSUE THREAD

**Issue by seanthepottingshed** [rainlab/builder-plugin#42]
Top work on the builder plugin peeps!

Would it be possible to add the import / export traits via the Builder plugin?

**Comment by alekseybobkov** [comment]
Yes, we will do it eventually. But it's considered to be a low priority task. Feel free to post a pull request implementing this feature. Thanks.

**Comment by seanthepottingshed** [comment]
Cheers, I will try and have a crack at it

**Comment by seanthepottingshed** [comment]
Hi @alekseybobkov 

I've got so far with implementing this feature. However, I've never done a pull request before and I've also got a little stuck with the following error when accessing Import page:

`Call to undefined method October\Rain\Database\QueryBuilder::getImportFilePath()`

How should I proceed in order to resolve this?

**Comment by seanthepottingshed** [comment]
OK, sorted the above out, however I don't know how to refresh the model list ( my solution creates scaffolds for the `Import` and `Export` models ) after I create the necessary models, so you have to refresh the page for them to appear. Any ideas?

Finally still clueless as to how to perform a pull request :(

**Comment by pvullioud** [comment]
hi @seanthepottingshed 
here is a tutorial about Pull request : https://yangsu.github.io/pull-request-tutorial
I can also help you to do it. Feel free to contact me on github.

**Comment by seanthepottingshed** [comment]
@pvullioud Nice one, you are a LEGEND

**Comment by Alex360hd** [comment]
Any update about this ? Would be great to be able to facilitate the import / export process of a model with the builder plugin, like for reordering
